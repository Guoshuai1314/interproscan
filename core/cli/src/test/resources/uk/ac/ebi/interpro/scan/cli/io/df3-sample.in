pdb|101mA	3_2_0:2z8aA00	154	0	146	136	1	143	1	146	2.9e-28	97.00	0.00	1	1:143
pdb|101mA	3_2_0:2dn1B00	154	0	153	146	1	148	1	146	5.5e-34	115.00	0.00	1	1:148
pdb|101mA	3_2_0:1it2A00	154	0	145	135	1	147	1	146	1.2e-33	109.80	0.00	1	1:147
pdb|101mA	3_2_0:1cg5B00	154	0	148	141	1	148	1	141	2.1e-30	103.80	0.00	1	1:148
pdb|101mA	3_2_0:1gdjA00	154	0	160	148	1	151	1	153	1.9e-24	84.40	0.00	1	1:151
pdb|101mA	3_2_0:1hlbA00	154	0	155	146	1	148	1	158	1e-29	103.00	0.00	1	1:148
pdb|101mA	3_2_0:1h97A00	154	0	154	141	2	147	1	147	2.4e-34	119.70	0.00	1	2:147
pdb|101mA	3_2_0:1irdA00	154	0	144	141	2	148	1	141	7.9e-34	115.30	0.00	1	2:148
pdb|101mA	3_2_0:1jf3A00	154	0	154	140	2	149	1	147	5.3e-27	94.10	0.00	1	2:149
pdb|101mA	3_2_0:1b0bA00	154	0	149	139	2	147	1	142	2e-17	58.10	0.00	1	2:147
pdb|101mA	3_2_0:1mbaA00	154	0	153	143	2	148	1	146	2.7e-34	116.10	0.00	1	2:148
pdb|101mA	3_2_0:1cg5A00	154	0	144	141	2	148	1	141	1.4e-35	123.00	0.00	1	2:148
pdb|101mA	3_2_0:1cqxA01	154	0	150	140	2	153	1	150	5.3e-20	66.70	0.00	1	2:153
pdb|101mA	3_2_0:1urvA00	154	0	161	151	2	152	1	154	5.1e-35	119.50	0.00	1	2:152
pdb|101mA	3_2_0:1ithA00	154	0	144	139	2	144	1	141	5.5e-27	94.10	0.00	1	2:144
pdb|101mA	3_2_0:1q1fA00	154	0	158	147	3	150	1	151	2.5e-26	91.50	0.00	1	3:150
pdb|101mA	3_2_0:1ecaA00	154	0	140	136	3	147	1	136	1.9e-29	102.00	0.00	1	3:147
pdb|101mA	3_2_0:1ashA00	154	0	157	143	3	148	1	150	4e-16	55.90	0.00	1	3:148
pdb|101mA	3_2_0:2nrlA00	154	0	145	141	5	148	1	141	6.4e-35	123.30	0.00	1	5:148
pdb|101mA	3_2_0:1ew6A00	154	0	141	134	10	149	1	137	1.7e-25	95.40	0.00	1	10:149
pdb|101mA	3_2_0:1kr7A00	154	0	114	110	12	143	1	110	1.3e-15	53.80	0.00	1	12:143