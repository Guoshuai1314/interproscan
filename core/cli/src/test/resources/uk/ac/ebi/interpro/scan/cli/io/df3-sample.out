pdb|101mA	3_2_0:1cg5A00	154	0	0	0	0	0	0	0	1.4e-35	0.00	0.00	1	2:148	 1
pdb|103lA	3_2_0:1swyA00	159	0	0	0	0	0	0	0	1e-50	0.00	0.00	1	1:159	 1
pdb|10mhA	3_2_0:1g55A02	327	0	0	0	0	0	0	0	2.2e-60	0.00	0.00	1	10:195	 1
pdb|10mhA	3_2_0:2c7pA02	327	0	0	0	0	0	0	0	2e-32	0.00	0.00	1	196:284	 1
pdb|11asA	3_2_0:12asA00	327	0	0	0	0	0	0	0	1e-139	0.00	0.00	1	1:327	 1
pdb|11baA	3_2_0:1eicA00	124	0	0	0	0	0	0	0	8.7e-52	0.00	0.00	1	1:124	 1
pdb|125d0	3_2_0:1d66B01	43	0	0	0	0	0	0	0	5.2e-22	0.00	0.00	1	1:40	 1
pdb|12e8H	3_2_0:1mfaH01	221	0	0	0	0	0	0	0	3.5e-48	0.00	0.00	1	1:96	 1
pdb|12e8H	3_2_0:1q9rB02	221	0	0	0	0	0	0	0	1.2e-27	0.00	0.00	1	121:219	 1
pdb|12e8L	3_2_0:2hp4A00	214	0	0	0	0	0	0	0	6.8e-40	0.00	0.00	1	2:107	 1
pdb|12e8L	3_2_0:2bnuB02	214	0	0	0	0	0	0	0	2e-30	0.00	0.00	1	108:214	 1
pdb|135lA	3_2_0:2vb1A00	129	0	0	0	0	0	0	0	1e-51	0.00	0.00	1	1:129	 1
pdb|13gsA	3_2_0:2c4jA01	210	0	0	0	0	0	0	0	1.2e-32	0.00	0.00	1	2:93	 1
pdb|13gsA	3_2_0:2a2rA02	210	0	0	0	0	0	0	0	4.6e-37	0.00	0.00	1	94:187	 1
pdb|13pkA	3_2_0:1zmrA02	415	0	0	0	0	0	0	0	1.1e-76	0.00	0.00	1	187:404	 1
pdb|13pkA	3_2_0:16pkA01	415	0	0	0	0	0	0	0	7.5e-68	0.00	0.00	1	1:189	 1
pdb|151c0	3_2_0:351cA00	82	0	0	0	0	0	0	0	4.5e-29	0.00	0.00	1	1:82	 1
pdb|153lA	3_2_0:1gbsA00	185	0	0	0	0	0	0	0	2e-62	0.00	0.00	1	1:185	 1
pdb|155cA	3_2_0:2bh4X00	134	0	0	0	0	0	0	0	5.5e-36	0.00	0.00	1	1:133	 1
pdb|156b0	3_2_0:156b000	103	0	0	0	0	0	0	0	5.8e-34	0.00	0.00	1	1:103	 1
pdb|15c8H	3_2_0:1mfaH01	217	0	0	0	0	0	0	0	2.3e-51	0.00	0.00	1	1:97	 1
pdb|15c8H	3_2_0:1q9rB02	217	0	0	0	0	0	0	0	8.9e-27	0.00	0.00	1	120:216	 1
pdb|16vpA	3_2_0:16vpA00	311	0	0	0	0	0	0	0	1.3e-119	0.00	0.00	1	1:308	 1
pdb|1914A	3_2_0:1914A00	171	0	0	0	0	0	0	0	4.9e-63	0.00	0.00	1	1:169	 1
pdb|1a02F	3_2_0:1a02F00	53	0	0	0	0	0	0	0	1.5e-09	0.00	0.00	1	1:52	 1
pdb|1a02J	3_2_0:1sknP00	52	0	0	0	0	0	0	0	2.2e-07	0.00	0.00	1	1:22	 1
pdb|1a02J	3_2_0:1junA00	52	0	0	0	0	0	0	0	3.5e-11	0.00	0.00	1	23:52	 1
pdb|1a03A	3_2_0:1k8uA00	90	0	0	0	0	0	0	0	8e-28	0.00	0.00	1	1:90	 1
pdb|1a04A	3_2_0:2qsjB00	205	0	0	0	0	0	0	0	1e-35	0.00	0.00	1	1:127	 1
pdb|1a04A	3_2_0:1je8E00	205	0	0	0	0	0	0	0	5.4e-23	0.00	0.00	1	128:205	 1
pdb|1a05A	3_2_0:1cnzA00	357	0	0	0	0	0	0	0	4e-135	0.00	0.00	1	1:354	 1
pdb|1a06A	3_2_0:2fstX01	279	0	0	0	0	0	0	0	1.6e-30	0.00	0.00	1	1:96	 1
pdb|1a06A	3_2_0:1jksA02	279	0	0	0	0	0	0	0	2e-52	0.00	0.00	1	97:255	 1
pdb|1a0aA	3_2_0:1a0aA00	63	0	0	0	0	0	0	0	2.9e-18	0.00	0.00	1	1:63	 1
pdb|1a0cA	3_2_0:1a0cA00	437	0	0	0	0	0	0	0	1.8e-193	0.00	0.00	1	1:437	 1
pdb|1a0dA	3_2_0:1a0cA00	437	0	0	0	0	0	0	0	2.8e-192	0.00	0.00	1	1:436	 1
pdb|1a0eA	3_2_0:1a0cA00	443	0	0	0	0	0	0	0	5.3e-193	0.00	0.00	1	1:438	 1
pdb|1a0fA	3_2_0:2pvqA01	201	0	0	0	0	0	0	0	1.3e-26	0.00	0.00	1	1:89	 1
pdb|1a0fA	3_2_0:2pvqA02	201	0	0	0	0	0	0	0	2e-34	0.00	0.00	1	90:188	 1
pdb|1a0gB	3_2_0:3daaA02	282	0	0	0	0	0	0	0	5.6e-52	0.00	0.00	1	119:277	 1
pdb|1a0gB	3_2_0:3daaA01	282	0	0	0	0	0	0	0	1.4e-35	0.00	0.00	1	1:119	 1
pdb|1a0hA	3_2_0:1a0hA02	159	0	0	0	0	0	0	0	3e-29	0.00	0.00	1	76:159	 1
pdb|1a0hA	3_2_0:2pf2A00	159	0	0	0	0	0	0	0	3.1e-29	0.00	0.00	1	2:89	 1
pdb|1a0iA	3_2_0:1a0iA03	332	0	0	0	0	0	0	0	7.4e-78	0.00	0.00	1	36:174	 1
pdb|1a0iA	3_2_0:1fviA01	332	0	0	0	0	0	0	0	1.3e-07	0.00	0.00	1	175:230	 1
pdb|1a0iA	3_2_0:1a0iA02	332	0	0	0	0	0	0	0	1.4e-44	0.00	0.00	1	232:332	 1
pdb|1a0jA	3_2_0:2aiqA02	223	0	0	0	0	0	0	0	4.2e-54	0.00	0.00	1	11:112	 1
pdb|1a0jA	3_2_0:1dleB02	223	0	0	0	0	0	0	0	3.2e-42	0.00	0.00	1	113:217	 1
pdb|1a0kA	3_2_0:3nulA00	130	0	0	0	0	0	0	0	9.6e-51	0.00	0.00	1	1:130	 1
pdb|1a0lA	3_2_0:2aiqA02	244	0	0	0	0	0	0	0	1.5e-53	0.00	0.00	1	11:120	 1
pdb|1a0lA	3_2_0:1dleB02	244	0	0	0	0	0	0	0	2.7e-44	0.00	0.00	1	121:238	 1
pdb|1a0oB	3_2_0:1eayD00	70	0	0	0	0	0	0	0	2e-20	0.00	0.00	1	3:70	 1
pdb|1a0pA	3_2_0:1a0pA02	271	0	0	0	0	0	0	0	7.5e-56	0.00	0.00	1	92:271	 1
pdb|1a0pA	3_2_0:1a0pA01	271	0	0	0	0	0	0	0	9.7e-25	0.00	0.00	1	1:91	 1
pdb|1a0rP	3_2_0:1a0rP02	188	0	0	0	0	0	0	0	1.2e-20	0.00	0.00	1	2:54	 1
pdb|1a0rP	3_2_0:2trcP01	188	0	0	0	0	0	0	0	2.7e-37	0.00	0.00	1	54:188	 1
pdb|1a0sP	3_2_0:1a0sP00	413	0	0	0	0	0	0	0	2.2e-143	0.00	0.00	1	1:413	 1
pdb|1a10E	3_2_0:2pwaA00	274	0	0	0	0	0	0	0	5.5e-88	0.00	0.00	1	6:274	 1
pdb|1a12A	3_2_0:1i2mB00	401	0	0	0	0	0	0	0	2.2e-108	0.00	0.00	1	4:401	 1
pdb|1a17A	3_2_0:1ihgA02	159	0	0	0	0	0	0	0	4.9e-27	0.00	0.00	1	8:127	 1
pdb|1a1mA	3_2_0:1zs8A01	278	0	0	0	0	0	0	0	2.1e-88	0.00	0.00	1	2:181	 1
pdb|1a1mA	3_2_0:1u58A02	278	0	0	0	0	0	0	0	8.7e-34	0.00	0.00	1	182:277	 1
pdb|1a1sA	3_2_0:1zq6A02	313	0	0	0	0	0	0	0	3.8e-74	0.00	0.00	1	137:296	 1
pdb|1a1sA	3_2_0:1js1Y01	313	0	0	0	0	0	0	0	2e-59	0.00	0.00	1	6:157	 1
pdb|1a1tA	3_2_0:1a1tA00	55	0	0	0	0	0	0	0	5e-18	0.00	0.00	1	1:55	 1
pdb|1a1wA	3_2_0:1n3kA00	83	0	0	0	0	0	0	0	1.8e-24	0.00	0.00	1	1:83	 1
pdb|1a21B	3_2_0:2hftA01	197	0	0	0	0	0	0	0	1.8e-32	0.00	0.00	1	4:102	 1
pdb|1a21B	3_2_0:2a2qT02	197	0	0	0	0	0	0	0	7.2e-39	0.00	0.00	1	103:197	 1
pdb|1a23A	3_2_0:1fvkA00	189	0	0	0	0	0	0	0	5.9e-63	0.00	0.00	1	1:189	 1
pdb|1a25A	3_2_0:2cm5A00	132	0	0	0	0	0	0	0	1.1e-36	0.00	0.00	1	1:125	 1
pdb|1a26A	3_2_0:1efyA02	351	0	0	0	0	0	0	0	2.8e-92	0.00	0.00	1	137:349	 1
pdb|1a26A	3_2_0:1efyA01	351	0	0	0	0	0	0	0	2.5e-54	0.00	0.00	1	1:136	 1
pdb|1a27A	3_2_0:1jtvA00	285	0	0	0	0	0	0	0	1.3e-65	0.00	0.00	1	1:267	 1
pdb|1a28A	3_2_0:2ax6A00	251	0	0	0	0	0	0	0	6.8e-71	0.00	0.00	1	1:251	 1
pdb|1a2aA	3_2_0:1mc2A00	122	0	0	0	0	0	0	0	4.2e-47	0.00	0.00	1	1:122	 1
pdb|1a2iA	3_2_0:1up9A00	107	0	0	0	0	0	0	0	3.7e-19	0.00	0.00	1	1:107	 1
pdb|1a2oA	3_2_0:1chdA00	347	0	0	0	0	0	0	0	4.8e-71	0.00	0.00	1	147:342	 1
pdb|1a2oA	3_2_0:1a2oA01	347	0	0	0	0	0	0	0	6e-28	0.00	0.00	1	1:142	 1
pdb|1a2sA	3_2_0:1ctjA00	89	0	0	0	0	0	0	0	1.4e-28	0.00	0.00	1	1:89	 1
pdb|1a2tA	3_2_0:1ey0A00	134	0	0	0	0	0	0	0	1.1e-29	0.00	0.00	1	2:134	 1
pdb|1a2xB	3_2_0:1a2xB00	31	0	0	0	0	0	0	0	9.5e-10	0.00	0.00	1	1:31	 1
pdb|1a2yB	3_2_0:1mfaH01	116	0	0	0	0	0	0	0	3.4e-49	0.00	0.00	1	1:96	 1
pdb|1a2zA	3_2_0:1iu8A00	220	0	0	0	0	0	0	0	5.5e-60	0.00	0.00	1	2:215	 1
pdb|1a30A	3_2_0:2hs1A00	99	0	0	0	0	0	0	0	1.4e-50	0.00	0.00	1	1:99	 1
pdb|1a31A	3_2_0:1a31A03	457	0	0	0	0	0	0	0	5.1e-67	0.00	0.00	1	217:366	 1
pdb|1a31A	3_2_0:1oisA02	457	0	0	0	0	0	0	0	1.3e-63	0.00	0.00	2	1:17:109:217	 1
pdb|1a31A	3_2_0:1oisA01	457	0	0	0	0	0	0	0	7.5e-38	0.00	0.00	1	18:105	 1
pdb|1a31A	3_2_0:1rr8C04	457	0	0	0	0	0	0	0	6.7e-33	0.00	0.00	1	372:457	 1
pdb|1a32A	3_2_0:2gybO00	85	0	0	0	0	0	0	0	1.4e-35	0.00	0.00	1	1:85	 1
pdb|1a34A	3_2_0:1a34A00	147	0	0	0	0	0	0	0	6.9e-93	0.00	0.00	1	1:147	 1
pdb|1a39A	3_2_0:1gpiA00	401	0	0	0	0	0	0	0	3.6e-148	0.00	0.00	1	5:401	 1
pdb|1a3aC	3_2_0:1a3aC00	146	0	0	0	0	0	0	0	9.7e-41	0.00	0.00	1	5:144	 1
pdb|1a3dA	3_2_0:1le6A00	119	0	0	0	0	0	0	0	1.3e-48	0.00	0.00	1	1:119	 1
pdb|1a3lH	3_2_0:1mfaH01	218	0	0	0	0	0	0	0	4.3e-47	0.00	0.00	1	1:97	 1
pdb|1a3lH	3_2_0:1q9rB02	218	0	0	0	0	0	0	0	1.2e-27	0.00	0.00	1	120:218	 1
pdb|1a3lL	3_2_0:1mjuL01	217	0	0	0	0	0	0	0	1.6e-35	0.00	0.00	1	1:112	 1
pdb|1a3lL	3_2_0:2bnuB02	217	0	0	0	0	0	0	0	4.6e-30	0.00	0.00	1	113:217	 1
pdb|1a3pA	3_2_0:1k36A00	43	0	0	0	0	0	0	0	8.3e-10	0.00	0.00	1	3:42	 1
pdb|1a3rL	3_2_0:2hp4A00	220	0	0	0	0	0	0	0	2.2e-39	0.00	0.00	1	2:113	 1
pdb|1a3rL	3_2_0:2bnuB02	220	0	0	0	0	0	0	0	1.3e-30	0.00	0.00	1	114:220	 1
